devito/dle/blocking_utils.py

```diff
@@ -51,20 +59,20 @@ def fold_blockable_tree(node, blockinner=True):
                 continue
             # Perform folding
             for j in pairwise_folds:
-                root, remainder = j[0], j[1:]
-                folds = [(tuple(y-x for x, y in zip(i.offsets, root.offsets)), i.nodes)
+                r, remainder = j[0], j[1:]
```

Comment:
`root` is better than `r` imo

Reply:
it doesn't fit in the 90chars line limit which makes the below a bit ugly, hence `r`. By the way, ``root`` is actually quite misleading :( but I haven't found a better name yet...